src/commands/export.ts

```diff
@@ -193,8 +193,12 @@ export const doExport = async (uri: Uri, document: TextDocument) => {
           },
         )
 
+        const shouldOpen = marpConfiguration().get<boolean>('exportAutoOpen')!
+
         if (outputToLocalFS) {
-          env.openExternal(uri)
+          if (shouldOpen) {
+            env.openExternal(uri)
+          }
```

Comment:
```suggestion
        if (outputToLocalFS && shouldOpen) {
          env.openExternal(uri)
```

If the file is not opened automatically, I think it's better to run `else` clause to provide a feedback to the user about the completion of the export process.